### DISCUSSION TITLE
Add column Column Group support to TreeTable

### DISCUSSION THREAD

**Discussion by sergeydus** [primefaces/community#2166] [PrimeVue]
[PrimeReact already supports ColumnGroups in their TreeTable implementation](https://primereact.org/treetable/#column_group)
would be nice to have it available in primeVue aswell

**Comment by RuslanMakhmutov** [comment]
+1

**Reply by rkashapov-dfx** [reply]
+1

**Reply by slavic-dude** [reply]
+1

**Reply by RamilAsadullin** [reply]
+1

**Comment by MrMasimo** [comment]
+1

**Reply by iimamov** [reply]
+1

**Comment by RAIbrahim360** [comment]
@cagataycivici 1 year passed. Please add support

**Comment by OmskBuddy** [comment]
+1
A highly relevant feature
